src/functions-reference/bounded_discrete_distributions.Rmd

```diff
@@ -155,6 +155,132 @@ The log binomial probability mass of n successes in N trials given
 logit-scaled chance of success alpha dropping constant additive terms
 `r since("2.25")`
 
+## Binomial-logit generalized linear model (Logistic Regression) {#binomial-logit-glm}
+
+Stan also supplies a single function for a generalized linear model
+with Binomial likelihood and logit link function, i.e. a function
+for logistic regression with aggregated outcomes. This provides a more efficient
+implementation of logistic regression than a manually written
+regression in terms of a Binomial likelihood and matrix
+multiplication.
+
+### Probability mass function
+
+Suppose $N \in \mathbb{N}$, $x\in \mathbb{R}^{n\cdot m}, \alpha \in \mathbb{R}^n, \beta \in \mathbb{R}^m$, and $n \in
+\{0,\ldots,N\}$.  Then \begin{align*}
```

Comment:
Start the `begin{align*}` on its own line and line up the text under it so that it's readable. Looks like this accidentally got collapsed into a paragraph.

In the Stan doc, if we have an M x N matrix, we've conventionally used [m, n] for indexing, not [i, j].

Reply:
> In the Stan doc, if we have an M x N matrix, we've conventionally used [m, n] for indexing, not [i, j].

That's not the case for the `_glm` docs

Reply:
Then we should change the `_glm` docs to be consistent with the rest of the *User's Guide*.  I never wrote down hard and fast style rules and things like the symbol for expectation starts drifting under multiple authors.  I mean to go and do a consistency fix of the entire doc set soon.